report/Tools/Github.tex

```diff
@@ -25,4 +25,8 @@ \subsection{Git and GitHub within the application}
 
 Our application also allows the Product Owner to see communication between developers, and intervene and make sure that the developers are on track.
 
-And since the application is already integrated with GitHub it was easy to implement a way for the Product Owner to be updated on the progress without the developers having to actively do so. This is done by having the developer update the labels on the issue on GitHub, and the results in the application showing the status of the different issues. 
+And since the application is already integrated with GitHub it was easy to implement a way for the Product Owner to be updated on the progress without the developers having to actively do so.
+This is done by having the developer update the labels on the issue on GitHub, and the results in the application showing the status of the different issues. 
+
+In this project we only implemented GitHub, since it is something we are very familiar with, but any alternative to GitHub such as BitBucket, Phabricator, GitLab, Atlassian, SourceForge or Launchpad would work just as well. However they each would require specific implementation to be supported.
```

Comment:
```suggestion
In this project we only implemented GitHub, since it is something we are very familiar with, but any alternative to GitHub such as BitBucket, Phabricator, GitLab, Atlassian, SourceForge or Launchpad would work just as well. However, they each would require specific implementation to be supported.
```